gluestring/gluegun.py

```diff
@@ -4,9 +4,16 @@
     "default": "NA"
 }
 
+DEFAULT_DELIMITERS = {
+    "start": "{{",
+    "end": "}}",
+}
+
 
 class Gluegun:
-    def __init__(self, mapping=DEFAULT_DICTIONARY):
+
+    def __init__(self, mapping=DEFAULT_DICTIONARY, delimiters=DEFAULT_DELIMITERS):
```

Comment:
Instead of passing delimiters as a separate parameter can you make it generic like `options` and inside the `options` dictionary you can add custom delimiters, so that later any new configuration has to be added, we don't end up adding new parameters each time.

Reply:
Also make `options` optional so the old projects using the same function should not break
Run the command `python3 -m unittest` to run the test cases. Make sure there are no failures without updating the test case. Feel free to ask for help if you face any issues

Reply:
There are test cases failing, ensure that `options` is optional, thats causing the failure

Reply:
> `python3 -m unittest` to run the test cases

Opps, I forgot to run all the tests